dallinger/experiment_server/experiment_server.py

```diff
@@ -46,8 +46,13 @@
 
 
 config = get_config()
-if not config.ready:
-    config.load_config()
+try:
+    from dallinger_experiment import extra_settings
+except ImportError:
+    pass
+else:
+    extra_settings()
+config.load_config()
```

Comment:
@MatthewWilkes This broke `dallinger debug` because the config was already loaded to get settings for gunicorn before it imports this module. I added a workaround in ea75ef81136d1732aa6326b09719c5e9f7cc4481 to import this earlier.